backend/backend/views.py

```diff
@@ -72,6 +64,17 @@ def get(self, request, *args, **kwargs):
 
         return ApiResponse({"tokens": [_bearer_token_dict(token, include_payload=False) for token in tokens]})
 
+    def delete(self, request, *args, **kwargs):
+        if BearerToken.objects.filter(token_id=request.GET.get("id")):
+            token = BearerToken.objects.get(token_id=request.GET.get("id"))
+            if request.user == token.user:
+                token.delete()
+                return HttpResponse("Authorized, token removed", status=200)
+            else:
+                return HttpResponse("Unauthorized, token was NOT removed", status=401)
```

Comment:
I would recommend using `from rest_framework import status` (from rest_framework import status)  instead of plain number for the HTTP code, it increase the readability (we directly now if you return a UNAUTHORIZED/FORBIDDEN/OK/CONFLICT etc) and could be automatically auto-completed by your IDE.